### ISSUE TITLE
No error message is reported within the HTML page if the repo already exist

### ISSUE THREAD

**Issue by cmoulliard** [fabric8io/fabric8#4718]
No error message is reported within the HTML page if the repo already exist

![screenshot 2015-09-14 12 06 31](https://cloud.githubusercontent.com/assets/463790/9847451/290064b6-5ad9-11e5-9faf-a45128fe5b59.png)

**Comment by davsclaus** [comment]
You mean the project name `dummy` already exists?

**Comment by jstrachan** [comment]
yeah, if you pick a name which already has a project in gogs there's no UI check and then things barf
